### PR TITLE
Add mamonca

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@samwaseda Can you confirm that you want to join in maintaining this package?

**Comment by samwaseda** [comment]
Yes! but I also wanted to know how it works

**Comment by jan-janssen** [comment]
I basically just used https://www.marcelotrevisani.com/grayskull and entered the name of your package.

**Comment by jan-janssen** [comment]
```
  Traceback (most recent call last):
    File "<string>", line 2, in <module>
    File "<pip-setuptools-caller>", line 34, in <module>
    File "/home/conda/staged-recipes/build_artifacts/mamonca_1663745375813/work/setup.py", line 24, in <module>
      cmdclass = {'build_ext': my_build_ext}, ext_modules=cythonize("mc.pyx", language_level="3")
    File "/home/conda/staged-recipes/build_artifacts/mamonca_1663745375813/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/Cython/Build/Dependencies.py", line 970, in cythonize
      module_list, module_metadata = create_extension_list(
    File "/home/conda/staged-recipes/build_artifacts/mamonca_1663745375813/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/Cython/Build/Dependencies.py", line 816, in create_extension_list
      for file in nonempty(sorted(extended_iglob(filepattern)), "'%s' doesn't match any files" % filepattern):
    File "/home/conda/staged-recipes/build_artifacts/mamonca_1663745375813/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/Cython/Build/Dependencies.py", line 114, in nonempty
      raise ValueError(error_msg)
  ValueError: 'mc.pyx' doesn't match any files
  error: subprocess-exited-with-error
```

**Comment by jan-janssen** [comment]
@srmnitc to coordinate the creation of conda-forge recipes at @eisenforschung

**Comment by jan-janssen** [comment]
```
ValueError: License file given in about/license_file (/home/conda/staged-recipes-copy/recipes/mamonca/PLEASE_ADD_LICENSE_FILE) does not exist in source root dir or in recipe root dir (with meta.yaml)
```

**Comment by jan-janssen** [comment]
Added an empty license file until the code is correctly licensed https://github.com/samwaseda/sams-studio/issues/4

**Comment by jan-janssen** [comment]
Mac OS X fails with:
```
 Error compiling Cython file:
  ------------------------------------------------------------
  ...

  from MC cimport MC as MCcpp
  from libcpp.vector cimport vector
  import numpy as np

  cdef class MC:
      ^
  ------------------------------------------------------------

  mc.pyx:7:5: 'MC' redeclared

  Error compiling Cython file:
  ------------------------------------------------------------
  ...

  cdef extern from "MC.cpp":
      pass

  cdef extern from "MC.h":
      cdef cppclass MC:
          ^
  ------------------------------------------------------------
```

File: recipes/mamonca/meta.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ requirements:
     - python
   run:
     - python
+    - numpy
```
Comment (jan-janssen) [inline recipes/mamonca/meta.yaml:30]: Update dependencies up-stream https://github.com/samwaseda/sams-studio/pull/6

**Comment by jan-janssen** [comment]
> Mac OS X fails with:
> 
> ```
>  Error compiling Cython file:
>   ------------------------------------------------------------
>   ...
> 
>   from MC cimport MC as MCcpp
>   from libcpp.vector cimport vector
>   import numpy as np
> 
>   cdef class MC:
>       ^
>   ------------------------------------------------------------
> 
>   mc.pyx:7:5: 'MC' redeclared
> 
>   Error compiling Cython file:
>   ------------------------------------------------------------
>   ...
> 
>   cdef extern from "MC.cpp":
>       pass
> 
>   cdef extern from "MC.h":
>       cdef cppclass MC:
>           ^
>   ------------------------------------------------------------
> ```

Same bug for windows: 
```
  Error compiling Cython file:
  ------------------------------------------------------------
  ...

  from MC cimport MC as MCcpp
  from libcpp.vector cimport vector
  import numpy as np

  cdef class MC:
      ^
  ------------------------------------------------------------

  mc.pyx:7:5: 'MC' redeclared

  Error compiling Cython file:
  ------------------------------------------------------------
  ...

  cdef extern from "MC.cpp":
      pass

  cdef extern from "MC.h":
      cdef cppclass MC:
```

**Pull request closed without merging**